src/main/resources/assets/farmersdelight/lang/it_it.json

```diff
@@ -192,7 +213,7 @@
 
   "itemGroup.farmersdelight": "Farmer's Delight",
 
-  "farmersdelight.container.recipe_book.cookable": "Mostra ciò che si può cuocere",
+  "farmersdelight.container.recipe_book.cookable": "Mostra cio' che si puo' cuocere",

```

Comment:
Yeah, the base font can support accented letters already, so this change isn't necessary.
![image](https://github.com/user-attachments/assets/92258a39-0d50-47a0-9f10-e0ce7a9126a6)
Could you please revert the accent changes?